CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## v1.8.0
+
+This release upgrades gcc from version 7.3.0 to 8.3.0. See the toolchain release
+notes for more information.
+
+* Enhancements
+  * Enable source-based routing for [vintage_net](https://github.com/nerves-networking/vintage_net)
```

Comment:
"Enable source-based routing in the Linux kernel to support [vintage_net](https://github.com/nerves-networking/vintage_net)"